dev/AppNotifications/AppNotificationBuilder/AppNotificationComboBox.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationComboBox.g.h"
+
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    struct AppNotificationComboBox : AppNotificationComboBoxT<AppNotificationComboBox, winrt::Windows::Foundation::IStringable>
+    {
+        AppNotificationComboBox(winrt::hstring const& id);
+
+        // Properties
+        void Items(winrt::Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> const& value) { m_items = value; };
+        winrt::Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> Items() { return m_items; };
+
+        void Title(winrt::hstring const& value) { m_title = value; };
+        winrt::hstring Title() { return m_title; };
+
+        void SelectedItem(winrt::hstring const& value) { m_selectedItem = value; };
+        winrt::hstring SelectedItem() { return m_selectedItem; };
+
+        // Fluent Setters
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox AddItem(winrt::hstring const& id, winrt::hstring const& content);
+
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox SetTitle(winrt::hstring const& value);
+
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox SetSelectedItem(winrt::hstring const& id);
+
+        // IStringable
+        winrt::hstring ToString();
+
+    private:
+        winrt::hstring m_id;
```

Comment:
Does winrt::hstring default to empty?

Reply:
Yup. Just tested this.

Reply:
Added empty initializers anyways as this is what the other classes in this module do.